python/pyspark/ml/tests.py

```diff
@@ -1572,7 +1588,8 @@ def test_java_params(self):
             for name, cls in inspect.getmembers(module, inspect.isclass):
                 if not name.endswith('Model') and issubclass(cls, JavaParams)\
                         and not inspect.isabstract(cls):
-                    self.check_params(cls())
+                    # NOTE: disable check_params_exist until there is parity with Scala API
+                    ParamTests.check_params(self, cls(), check_params_exist=False)
```

Comment:
Yes, ideally but most of the models need to be trained first so that is why they are skipped here.  Some basic framework would need to be added to allow this, and I'm looking into that as a follow on.